webapp/src/ts/modules/about/about.component.html

```diff
@@ -1,127 +1,182 @@
 <div class="row filters"></div>
```

Comment:
This line adds unnecessary gray line
<img width="500" src="https://github.com/medic/cht-core/assets/66472237/f6832fce-e870-4f64-a61e-ace6850336db">

It looks better without `<div class="row filters"></div>`
<img width="500" src="https://github.com/medic/cht-core/assets/66472237/d0c1b5f4-ebb1-4c8e-882e-d290eeb5af07">